docs/quickstart.md

```diff
@@ -0,0 +1,209 @@
+# Quickstart
```

Comment:
This is a little more involved than what I'd call a quickstart. In the first sentence this page is described as a "guided tour" which I think is more accurate to the current page.

Reply:
I changed it to being called a "guided tour".